### PR TITLE
Fix regression when parent list of a Content field had a `where` clau…

### PR DESCRIPTION
**Pull request by timleslie**
…se as its access control.

### CODE REVIEW

File: packages/fields/src/types/OEmbed/OEmbedBlock.js [hunk 1]
```diff
@@ -51,7 +51,16 @@ export class OEmbedBlock extends Block {
               'A reference back to the Slate.js Serialised Document this embed is contained within',
           },
         },
-        access: listConfig.listAccess,
+        access: Object.entries(listConfig.listAccess).reduce(
```
Comment (MadeByMike) [inline packages/fields/src/types/OEmbed/OEmbedBlock.js]: Util function?

Comment (MadeByMike) [reply packages/fields/src/types/OEmbed/OEmbedBlock.js]: It's now used 3 times?

Comment (MadeByMike) [reply packages/fields/src/types/OEmbed/OEmbedBlock.js]: 4 times!

Comment (timleslie) [reply packages/fields/src/types/OEmbed/OEmbedBlock.js]: Yeah, maybe, but also I don't have time right now :-)

Comment (timleslie) [reply packages/fields/src/types/OEmbed/OEmbedBlock.js]: The real refactor should come one level higher with the way we create these Aux lists for blocks, but that's future Tim's problem.

**Pull request merged**